app/views/issues/map.html.slim

```diff
@@ -0,0 +1,54 @@
+link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css"
```

Comment:
maybe create content_for :css and content_for JS and load these pages there? I don't like idea of adding js/css inside body